install/helm/space-agon/templates/frontend.yaml

```diff
@@ -12,6 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: frontend-sa
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    iam.gke.io/gcp-service-account: om-caller@peteryizhong-gke-dev.iam.gserviceaccount.com
```

Comment:
(same comment)
Could you replace this specific value to helm chart variable and add values.yaml?

For example:
https://github.com/googleforgames/agones/blob/v1.43.0/install/helm/agones/templates/serviceaccounts/controller.yaml#L28

Reply:
Done, I used `YOUR_GOOGLE_SERVICE_ACCOUNT` as a place holder because it looks more intuitive.

Reply:
Thank you. I could confirm `.Values.frontend.googleServiceAccount`.